build-config/make/crosstool-ng.make

```diff
@@ -9,16 +10,26 @@
 # This is a makefile fragment that defines the build of crosstool-NG
 #
 
+# Default GCC version to build for the toolchain
+GCC_VERSION 			?= 4.9.2
+XTOOLS_LIBC 			?= uClibc
+XTOOLS_LIBC_VERSION 		?= 0.9.33.2
+
 CROSSTOOL_NG_DESC		= crosstool-NG
-CROSSTOOL_NG_VERSION		= 1.21.0
-CROSSTOOL_NG_TARBALL		= crosstool-ng-$(CROSSTOOL_NG_VERSION).tar.xz
-CROSSTOOL_NG_URLS		+= $(ONIE_MIRROR) http://crosstool-ng.org/download/crosstool-ng
+CROSSTOOL_NG_VERSION		?= custom
```

Comment:
This is a great implementation.  It gives us a knob to turn.

My only quibble is that "custom" should be the exception, not the rule.  In other words the default should be:
```
CROSSTOOL_NG_VERSION    ?= 1.21.0
```

If a particular machine needs a custom setting that can be stored in the machine specific machine.mk file.

Another approach is that `CROSSTOOL_NG_VERSION` could be set to `custom` in `build-config/arch/armv8a.make`.  Then the custom setting would be the default for all armv8a based machines.  That may be the best approach here.  We do not have any armv8a machines in the repo yet, so now would be a good time to make that choice.